### ISSUE TITLE
Add support for Somfy Smoove Origin IO

### ISSUE THREAD

**Issue by DjBac** [iMicknl/ha-tahoma#389]
I cannot find a way to add my Somfy Remotes to HA. I can see my devices, scenes and even the alarm button of the hub onto HA, but no way to see the remotes. Is this supported?

**Comment by vlebourl** [comment]
Hi,
Please follow the templates when opening an issue. 
Remotes aren't supported yet. We didn't find any use that would justify adding it to Home Assistant. Could you explain your use case? 
Cheers.

**Comment by DjBac** [comment]
Hi,
Thank you for the reply. The only use would be to have notifications for battery status and use the remotes as buttons for Home Assistant.

**Comment by iMicknl** [comment]
@DjBac could you create a new issue please, where you fill in all questions of the predefined template? This is required to understand more about your device. I will close this one, please create a new one with all the details. In our README (https://github.com/iMicknl/ha-tahoma#device-not-supported) you can find how to gather those missing details.

However, due to the limitations by Somfy, we are only allowed to pull the new events every 30 seconds. This means that it can take up to 30 seconds before we can respond to a button press.. (see https://github.com/iMicknl/ha-tahoma/issues/259).
